modules/nf-core/tcoffee/align/meta.yml

```diff
@@ -0,0 +1,73 @@
+name: "tcoffee_align"
+description: Aligns sequences using T_COFFEE
+keywords:
+  - alignment
+  - MSA
+  - genomics
+tools:
+  - "famsa":
```

Comment:
Was about to comment but it is already merged, I think this is a typo and should be tcoffee